config_defaults.yml

```diff
@@ -29,13 +29,25 @@
     env: PUPPET_GEM_VERSION="~> 3.0"
     bundler_args: --without system_tests
 Gemfile:
+  supports_windows:
+    - puppetlabs-acl
+    - puppetlabs-chocolatey
+    - puppetlabs-dism
+    - puppetlabs-dsc
+    - puppetlabs-powershell
+    - puppetlabs-reboot
+    - puppetlabs-registry
+    - puppetlabs-sqlserver
+    - puppetlabs-wsus_client
```

Comment:
@glennsarti Both @ferventcoder and I had a discussion in hipchat yesterday about how `.sync.yml` should be centralized and go away entirely, but that is a project for another time.

In the mean time, it is modulesync_configs that cares about which modules receive Gemfile template variants, and the modules themselves should be ignorant of "fiddly bits" inside of the templates anyway, so the quality of extra template contents should live with the templates themselves.